python/cudf/setup.py

```diff
@@ -3,8 +3,9 @@
 from setuptools import find_packages
 from skbuild import setup
 
+packages = find_packages(include=["cudf*", "udf_cpp*"])
 setup(
-    include_package_data=True,
-    packages=find_packages(include=["cudf", "cudf.*"]),
+    packages=packages,
+    package_data={key: ["*.pxd", ".hpp", ".cuh"] for key in packages},
```

Comment:
Why does `pxd` have a `*` but the other extensions do not?

Reply:
According to docs, I think you want to change this. https://setuptools.pypa.io/en/latest/userguide/datafiles.html#package-data
```suggestion
    package_data={key: ["*.pxd", "*.hpp", "*.cuh"] for key in packages},
```

Reply:
Because typos. Not intentional. Also half the reason I changed this to draft while I fixed haha.